src/helpers/user.js

```diff
@@ -34,5 +34,6 @@ export function fromMtoVM(user) {
         }))
       : [],
     tags: user.tags ? user.tags.map(i => ({ label: i, value: i })) : [],
+    title: user.title ?? ''
```

Comment:
Have you faced with a specific issue here? `title` shouldn't be `undefined` / `null`

Reply:
> Have you faced with a specific issue here? `title` shouldn't be `undefined` / `null`

My user doesn't have a title which causes the uncontrolled input component to show up.

Reply:
No kidding. It's a problem in the API, it shouldn't return an empty string. 
Also the title is mandatory, how your title is undefined?